prometheus/common/alert-rules.d/crunchy-alert-rules-patroni.yml.example

```diff
@@ -0,0 +1,73 @@
+###
+#
+# Copyright © 2017-2022 Crunchy Data Solutions, Inc. All Rights Reserved.
+#
+###
+
+groups:
+- name: alert-rules
+  rules:
+
+########## PATRONI RULES ##########
+
+  - alert: PatroniPGIsRunning
+    expr: patroni_postgres_running != 1
+    for: 60s
+    labels:
+      service: patroni 
+      severity: critical
+      severity_num: 300
+    annotations:
+      summary: 'Patroni is reporting that PostgreSQL is not running on {{ $labels.job }}'
+
+  - alert: PatroniPendingRestart
+    expr: patroni_pending_restart != 0
+    for: 60s
+    labels:
+      service: patroni 
+      severity: critical
+      severity_num: 300
+    annotations:
+      summary: 'Patroni is reporting a pending restart on {{ $labels.job }}'
+
+  - alert: PatroniIsPaused
+    expr: patroni_is_paused != 0
+    for: 60s
+    labels:
+      service: patroni 
+      severity: critical
+      severity_num: 300
+    annotations:
+      summary: 'Patroni is reporting that it is in maintenance mode (paused) on {{ $labels.job }}'
+
+  - alert: PatroniDCSLastSeen
+    expr: (time() - patroni_dcs_last_seen) > 300
```

Comment:
This would depend on patroni TTL/Timeouts

Reply:
I was thinking that not communicating with the DCS for 5 minutes would be an issue, no? What are the normal timeouts here? If we make the alerts more directly configurable via ansible, could see about tying this to the timeout value. Otherwise, for now user is free to adjust these thresholds in their alert file they have to provide.